### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_Screenreader: Key bending status message "1 existing command has this key bending" is not announced by screen reader.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#7130]
--- **GitHub Tags:**
-A11yMAS; -A11yTCS; -A11ySev2; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2022; -WCAG4.1.3; -NVDA; -ScreenReader; -Narrator;

--- **Environment and OS details:**
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

--- **Repro Steps:**

1. Turn on NVDA
2. Open Visual studio code editor.
3. Tab till setting icon and Select "Keyboard shortcuts options"
4. Verify that Key bending status message "1 existing command has this key bending" is announced by screen reader or not.

--- **Actual Result:**
Key bending status message "1 existing command has this key bending" is not announced by NVDA/Narrator screen readers.

--- **Expected Result:**
NVDA/Narrator should announce the status message "1 existing command has this key bending" 

--- **Attachment:**
![NVDA](https://user-images.githubusercontent.com/91525824/187140805-1d1a28be-917a-4114-989c-533cd161fb10.gif)

![narrator](https://user-images.githubusercontent.com/91525824/187140950-65067465-9833-465a-bcee-663ea4a55ce8.gif)
